packages/core/src/types.ts

```diff
@@ -113,17 +113,16 @@ export interface DebugEventTypes {
   retryRetrying: {
     retryCount: number;
   };
-  // Graphcache Exchnage
-  graphcacheHit: {
-    policy: RequestPolicy;
-    value: OperationResult;
-  };
 }
 
-export type DebugEvent<T extends keyof DebugEventTypes | string> = {
+export type DebugEventArg<T extends keyof DebugEventTypes | string> = {
```

Comment:
Would have preferred to not require two DebugEvent types but this would not work

```ts
<T extends keyof DebugEventTypes | string>(arg: Omit<DebugEvent<T>, 'source'>) => void
```